### ISSUE TITLE
[pni] about page: don't skip heading levels

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4647]
Headings should not skip levels - particularly the methodology page has a lot of `<h3>` that should be `<h2>`
